nuxeo-features/nuxeo-platform-audit/nuxeo-platform-audit-core/src/main/java/org/nuxeo/ecm/platform/audit/provider/LatestCreatedUsersOrGroupsPageProvider.java

```diff
@@ -105,4 +111,13 @@ public long getResultsCountLimit() {
         return pp.getResultsCountLimit();
     }
 
+    protected boolean canSearchUsersAndGroups(NuxeoPrincipal principal) {
+        if (principal.isAdministrator() || principal.isMemberOf(POWER_USERS_GROUP)) {
+            return true;
+        }
+        UserManager um = Framework.getService(UserManager.class);
+        List<String> adminGroups = um.getAdministratorsGroups();
+        return adminGroups.stream().anyMatch(principal::isMemberOf);
```

Comment:
Isn't this redundant with `principal.isAdministrator()`, knowing that `UserManager#makePrincipal` goes through `NuxeoPrincipalImpl#setVirtualGroups` and `NuxeoPrincipalImpl#updateAllGroups`?

See https://github.com/nuxeo/nuxeo/blob/master/nuxeo-services/nuxeo-platform-usermanager-api/src/main/java/org/nuxeo/ecm/platform/usermanager/NuxeoPrincipalImpl.java#L367.